harvard-lts/mps-embed
# No https

**Remove https**
* * *


# What does this Pull Request do?
Removes https from embed internally and updates the healthcheck

# How should this be tested?

Spin up locally, see no https

A pr for the IF project will follow, TG need updating

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
